website/project/metadata/hypothesis-testing-studies-using-youth-data.json

```diff
@@ -0,0 +1,271 @@
+		"display_text": "Research questions",
+		"example_text": "Example: RQ1 = Are more religious people more prosocial than less religious people? RQ2 = Does the relationship between religiosity and prosociality differ for people with different religious affiliations?",
+		"required": false
+	}, {
+		"help_text": "",
+		"block_type": "long-text-input",
+		"display_text": "",
+		"example_text": "",
+		"required": true
+	}, {
+		"help_text": "For each of the research questions listed in the previous section, provide one or more specific and testable hypothesis. Please make clear whether the hypotheses are directional (e.g., A > B) or non-directional (e.g., A \u2260 B). If directional, state the direction. You may also provide a rationale for each hypothesis.",
+		"block_type": "question-label",
+		"display_text": "Hypotheses",
+		"example_text": "We expect that religiosity is associated with prosocial behavior in our sample as well. To assessthis prediction, we will use the following hypotheses:H0(1) = In men and women who graduated from Wisconsin high schools in 1957, there is no association between religiosity and prosociality H1(1) = In men and women who graduated from Wisconsin high schools in 1957, there is an association between religiosity and prosociality.It is possible that these mechanisms are stronger for some religions than for others.  and prosocial behavior differs for the different religions in our sample. To answer this question, we will test the following hypotheses: H0(2) = The relationship between religiosity and prosociality in men and women who graduated from Wisconsin high schools in 1957 does not differ for people with different religious affiliations (i.e., Roman Catholics, Lutheran, Methodists, Presbyterians, United Church of Christ, and Protestants) H1(2) = The relationship between religiosity and prosociality in men and women who graduated from Wisconsin high schools in 1957 differs for people with different religious affiliations (i.e., Roman Catholics, Lutheran, Methodists, Presbyterians, United Church of Christ, and Protestants).",
+		"required": false
+	}, {
+		"help_text": "",
+		"block_type": "long-text-input",
+		"display_text": "",
+		"example_text": "",
+		"required": true
+	}, {
+		"help_text": "",
+		"block_type": "page-heading",
+		"display_text": "Data Description",
+		"example_text": "",
+		"required": false
+	}, {
+		"help_text": "",
+		"block_type": "section-heading",
+		"display_text": "Data collection",
+		"example_text": "",
+		"required": true
+	}, {
+		"help_text": "For each of the research questions listed in the previous section, provide one or more specific and testable hypothesis. Please make clear whether the hypotheses are directional (e.g., A > B) or non-directional (e.g., A \u2260 B). If directional, state the direction. You may also provide a rationale for each hypothesis.",
+		"block_type": "question-label",
+		"display_text": "",
+		"example_text": "",
+		"required": true
```

Comment:
To be extra explicit, this whole block is a duplicate from the block in line 25 and should be deleted